install-and-configure/advanced-configuration/network-costs-configuration.md

```diff
@@ -199,6 +196,15 @@ networkCosts:
       #     - "10.0.0.0/24"
 ```
 
+## Permissions required
+
+The network-costs daemonset requires a privileged [`spec.containers[*].securityContext`](https://kubernetes.io/docs/concepts/security/pod-security-standards/) and `hostNetwork: true` in order to leverage an underlying kernel module to capture network data.
```

Comment:
How are we styling "network-costs"? Is it "network-costs", "network costs", "NetworkCosts"??? We need a standard.

Reply:
Oh wait, looks like @bstuder99 has already made the changes!

Reply:
indeed :)